metacat-connector-s3/src/main/java/com/netflix/metacat/connector/s3/S3Module.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.netflix.metacat.connector.s3;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.name.Names;
+import com.netflix.metacat.common.server.connectors.ConnectorDatabaseService;
+import com.netflix.metacat.common.server.connectors.ConnectorInfoConverter;
+import com.netflix.metacat.common.server.connectors.ConnectorPartitionService;
+import com.netflix.metacat.common.server.connectors.ConnectorTableService;
+import com.netflix.metacat.connector.s3.dao.DatabaseDao;
+import com.netflix.metacat.connector.s3.dao.FieldDao;
+import com.netflix.metacat.connector.s3.dao.PartitionDao;
+import com.netflix.metacat.connector.s3.dao.SourceDao;
+import com.netflix.metacat.connector.s3.dao.TableDao;
+import com.netflix.metacat.connector.s3.dao.impl.DatabaseDaoImpl;
+import com.netflix.metacat.connector.s3.dao.impl.FieldDaoImpl;
+import com.netflix.metacat.connector.s3.dao.impl.PartitionDaoImpl;
+import com.netflix.metacat.connector.s3.dao.impl.SourceDaoImpl;
+import com.netflix.metacat.connector.s3.dao.impl.TableDaoImpl;
+
+import java.util.Map;
+
+/**
+ * Guice module.
+ */
+public class S3Module implements Module {
+    private final String catalogName;
+    private final Map<String, String> configuration;
+    private final S3ConnectorInfoConverter infoConverter;
+
+    /**
+     * Constructor.
+     * @param catalogName catalog name.
+     * @param configuration configuration properties
+     * @param infoConverter S3 info converter
+     */
+    public S3Module(final String catalogName, final Map<String, String> configuration,
```

Comment:
Why pass in the info converter? isn't that going to be consistent across all instantiations of this module? I'd just build it in here.